chemprop/data/utils.py

```diff
@@ -242,6 +233,19 @@ def get_data(path: str,
             if len(all_smiles) >= max_data_size:
                 break
 
+        atom_features = None
+        atom_descriptors = None
+        if args is not None and args.atom_descriptors is not None:
+            try:
+                descriptors = load_valid_atom_features(atom_descriptors_path, [x[0] for x in all_smiles])
+            except Exception as e:
+                print('Error: failed to load or valid custom atomic descriptors: {}'.format(e))
```

Comment:
If this step fails, is it useful to continue (vs. raising the error and stopping)?

Also, I'm not sure what the convention in chemprop is for using print vs logging.

Reply:
Good point! It shouldn't continue. It should raise an error